_1327/main/templates/searched_list.html

```diff
@@ -0,0 +1,55 @@
+{% extends 'base_with_sidebar.html' %}
+{% load i18n %}
+
+{% block title %}
+	{% blocktrans count counter=2 %}Documents{% plural %}Documents{% endblocktrans %}
+{% endblock %}
+
+{% block sidebar %}
+	<div class="toc hidden-print">
+		<ul>
+			{% for type, documents in searched_documents %}
+				<li><a href="#type{{ type }}">{{ type }}</a></li>
+			{% endfor %}
+		</ul>
+	</div>
+{% endblock %}
+
+{% block content %}
+	{% for type, documents in searched_documents %}
+		<h3 id="type{{ type }}">{{ type }}</h3>
```

Comment:
please use `name="type{{ type }}"` here instead of `id`, just in case we might get such an id more than once on a page. This might break some things :sweat_smile: